debian/jitsi-meet-prosody.postinst

```diff
@@ -116,6 +115,18 @@ case "$1" in
             PROSODY_CONFIG_PRESENT="false"
         fi
 
+        # we always try to create the user 'jvb' and not fail configure if user exists
+        prosodyctl register jvb $JICOFO_AUTH_DOMAIN $JVB_SECRET || true
+
+        # Check whether prosody config has the internal muc, if not add it,
+        # as we are migrating configs
+        if ! grep -q "internal.auth.$JVB_HOSTNAME" $PROSODY_HOST_CONFIG; then
+            echo -e "\nComponent \"internal.auth.$JVB_HOSTNAME\" \"muc\"" >> $PROSODY_HOST_CONFIG
+            echo -e "    storage = \"null\"" >> $PROSODY_HOST_CONFIG
+            echo -e "    modules_enabled = { \"ping\"; }" >> $PROSODY_HOST_CONFIG
+            echo -e "    admins = { \"focusUser@auth.jitmeet.example.com\", \"jvb@auth.jitmeet.example.com\" }" >> $PROSODY_HOST_CONFIG
```

Comment:
Why do we need these users to be admins? Is it required for MUC creation?

Reply:
Yes exactly. It depends on who will start first and will create the room first, and must be admin.